src/main/java/com/uber/cadence/internal/common/WorkflowExecutionUtils.java

```diff
@@ -343,7 +333,7 @@ private static HistoryEvent getInstanceCloseEvent(
   }
 
   private static RetryOptions getRetryOptionWithTimeout(long timeout, TimeUnit unit) {
-    return new RetryOptions.Builder(retryParameters)
+    return new RetryOptions.Builder(RpcRetryer.DEFAULT_RPC_RETRY_OPTIONS)
```

Comment:
This line actually changes retry options, e.g. backoff coefficient is now 1.2 instead of previous 2, among others. Are we ok with this?

Reply:
Yeah I am aware of this and I think this is ok. There is no reason that this method is special, where backoff coefficient for it is 2 instead of 1.2. Better to keep it consistent everywhere.